data-pipeline/src/trace_exporter.rs

```diff
@@ -38,6 +41,13 @@ pub enum TraceExporterOutputFormat {
     V07,
 }
 
+// internal health metrics
+const STAT_SEND_TRACES: &str = "datadog.libdatadog.send.traces";
```

Comment:
These consts could probably be put into either an Enum or a separate module named `HealthMetrics` to better define what they represent. As it is right now, it's unclear for someone looking at the data-pipeline for the first time what they mean.